Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ruby:2.5.1
+RUN apt-get update -qq && apt-get upgrade -y && apt-get install -y build-essential libpq-dev libxml2-dev libxslt1-dev && apt-get clean
```

Comment:
I think it might be better to split the upgrade line and the installation line into two runs because this will allow Docker to cache the upgraded image and we can use it as a base when building other images (assuming the RUN line is the same string exactly).